src/stylesheets/elements/typography/_content.scss

```diff
@@ -45,14 +45,14 @@
   }
```

Comment:
Should the `<p>` and `<span>` selector in `usa-dark-background` get the reverse text color?

It's currently:
```scss
  p,
  span {
    color: color("white");
  }
```

Reply:
Made this change. The `span` seems unusual, but we'll leave it for now.